lib/styles/responsive_constants.dart

```diff
@@ -0,0 +1,4 @@
+class ResponsiveConstants {
+  static const double largeScreenHeight = 700;
+  static const double mediumScreenHeight = 500;
```

Comment:
@ahmed-tarek-salem  where is mediumScreenHeight being used?

Reply:
@sallymoc 
I used it first and then removed it from the code, resolved.